functional_tests/behave_fixtures.py

```diff
@@ -0,0 +1,87 @@
+import django
+from behave import fixture
+from behave.runner import Context
+from django.conf import settings
+from django.db import close_old_connections
+from django.test.runner import DiscoverRunner
+from django.test.testcases import LiveServerTestCase
+from dslr.config import settings as dslr_settings
+from dslr.operations import create_snapshot, delete_snapshot, find_snapshot, restore_snapshot
+
+# Adapted from https://behave.readthedocs.io/en/stable/usecase_django.html#manual-integration
+
+
+@fixture
+def django_test_runner(context: Context) -> None:
+    """A behave fixture to wrap the test run in a Django test runner.
+    This sets up a test database (prefixed with `test_`) so the test runs are isolated.
+    Saves a database dump to a file, to allow tests to reset the database to a clean, initial state.
+    Designed to be registered as a 'before_all' fixture, so that the setup occurs once before the tests,
+    and teardown follows after the entire run finishes.
+    """
+    # Set up Django, and it's test runner to give us a clean test database and initialised/set up Django resources
+    django.setup()
+    context.test_runner = DiscoverRunner()
+    context.test_runner.setup_test_environment()
+    context.old_db_config = context.test_runner.setup_databases()
+
+    # At this point, the Django test runner has initialised the test database and run migrations
+    # Initialize DSLR and take a snapshot of the clean, post migration test database
+    initialize_dslr()
+    context.clean_snapshot_name = "clean_snapshot"
```

Comment:
Shouldn't the comment and the function call be named `initialise_dslr`? 

Are you trying to be consistent with `dslr_settings.initialize(url=db_conn_str, debug=False)` code?

Reply:
I believe historically it's been a UK vs American English divide, but these days either spelling is considered accepted. I don't have strong feelings beyond maintaining consistency and within our code base, so I defaulted to the same spelling as they use in the DSLR module itself.

Reply:
The second part, yes thought so. Thanks!